### DISCUSSION TITLE
ParIS interaction

### DISCUSSION THREAD

**Discussion by AnaCarolinaChaves** [International-Data-Spaces-Association/DataspaceConnector#382] [General]
We have an infrastructure with all the IDS components, except the ParIS. The information available is that the DSC does not offer any interaction with this component.

Is there any plan to implement the Participant Update and Participant Unavailable Messages in the future?

**Comment by omarsilva1** [comment]
Hello Ana, we currently don't plan on adding ParIS support.

**Reply by AnaCarolinaChaves** [reply]
Thank you for the information.
So the information on the testbed repository is incorrect and there will be no implementation?

**Reply by omarsilva1** [reply]
Yes, the information is probably outdated.

**Reply by AnaCarolinaChaves** [reply]
Thank you.
